src/theme.ts

```diff
@@ -13,22 +13,43 @@ export const theme = {
     WHITE: "#FFFFFF",
   },
   hanehldaColors: {
-    CREAM: "#F5EBD5",
-    TEXT_CREAM: "#FFEA9F",
+    CREAM: "#fbf8f0",
     DARK_YELLOW: "#F7C85D",
-    DARK_BLUE: "#489ABD",
+    DARK_BLUE: "#6a9eb5",
     ERROR_RED: "#FFCED0",
-    LIGHT_RED: "#DFBFAD",
-    DARK_RED: "#98342E",
+    LIGHT_RED: "#c08176",
+    DARK_RED: "#a14738",
     BORDER_GRAY: "#D9D9D9",
-    DARK_GRAY: "#464D50",
+    DARK_GRAY: "#1a2327",
     LIGHT_GRAY: "#707070",
     TEXT_LIGHT_GRAY: "#898989",
-    LIGHT_GREEN: "#62B223",
+    LIGHT_GREEN: "#285c00",
     DARK_GREEN: "#0B455E",
+    
+    HEADERBAR: "#a14738",
+    WHITETEXT: "#ffffff",
+    DARK_BLUE_TEXT: "#0c455e",
+    DARK_RED_TEXT: "#390c05",
+    SIDEBAR: "#a94f3f",
+    LIGHT_YELLOW: "#f7ecc7",
+    TABLE1: "#f9f4e4",
+    TABLE2: "#fdfbf5",
+    WHITE_BUTTON: "#faf3f3",
+    WHITE_HIGHLIGHT: "#f4dba6",
```

Comment:
I see the appeal of all the `WHITE_...` styles going together, but I wonder if there is another name for this family that doesn't involve naming offwhite colors white. That feels like there's room for confusion down the line.